testsettings.py

```diff
@@ -15,6 +15,7 @@
 NOSE_ARGS = [
     #'--no-migrations' # trim ~120s from test run with db tests
     #'--with-fixture-bundling',
+    '--with-doctest',
```

Comment:
Then shouldn't we make sure we convert any doctests that were previously
being tested? Seems bad to just drop a bunch of tests